usr/sbin/p11sak/p11sak.c

```diff
@@ -978,6 +1046,15 @@ static CK_RV tok_key_list_init(CK_SESSION_HANDLE session, p11sak_kt kt,
                     p11_get_ckr(rc));
             return rc;
         }
+    } else if (kt == kt_ALL) {
+        rc = funcs->C_FindObjectsInit(session, NULL, 0);
+        if (rc != CKR_OK) {
+            printf("C_FindObjectInit failed\n");
+            printf("in tok_key_list_init() (error code 0x%lX: %s)\n", rc,
```

Comment:
Why two lines?  And why error messages to stdout?  Shouldn't they go to stderr?

Reply:
Done